api/app/models/land_change/tree_cover_gain.py

```diff
@@ -0,0 +1,105 @@
+from typing import Annotated, Optional, Union
+
+from app.models.common.analysis import AnalysisStatus, AnalyticsIn
+from app.models.common.areas_of_interest import (
+    AdminAreaOfInterest,
+    ProtectedAreaOfInterest,
+)
+from app.models.common.base import Response, StrictBaseModel
+from pydantic import Field, field_validator, model_validator
+
+AoiUnion = Union[
+    AdminAreaOfInterest,
+    ProtectedAreaOfInterest,
+]
```

Comment:
@jterry64, what AOIs are we going to support?

Reply:
We should support them all now, since I implemented OTF!